immuno_graphical/.gitignore

```diff
@@ -1,5 +1,6 @@
 data_clean/*.Rdata
 data_clean/*.rds
+data_clean/practice_data.csv
```

Comment:
this should not be where the `practice_data.csv` lives. it is already `.gitignore`d at the top level `data_clean` directory.